### DISCUSSION TITLE
Project demo videos.

### DISCUSSION THREAD

**Discussion by YuvrajRaghuvanshiS** [YuvrajRaghuvanshiS/WhatsApp-Key-Database-Extractor#33] [Ideas]
Hi there, I am looking for project demo videos to put in README of this project. It should be in gif format and should be as small as possible both in terms of length and size.
You can trim the video where it just waits for backup to complete or other time consuming parts.

**Comment by tpucci** [comment]
Excellent work !
I recorded my screen during script execution.

<details>
<summary>Mov file</summary>

https://user-images.githubusercontent.com/16262904/150645532-a23a8c58-8ea5-4849-bb34-621fd9be870e.mov 

</details>

![output](https://user-images.githubusercontent.com/16262904/150645504-8921b357-85c1-4fc2-87b0-8ca3ee019e78.gif)

**Reply by YuvrajRaghuvanshiS** [reply]
Thank you for this, I almost forgot about it.
